### ISSUE TITLE
[T4|P2] Variables inútiles

### ISSUE THREAD

**Issue by Drpinto1** [PUC-IIC2223/syllabus2019#30]
Hola :D

¿Podemos asumir que las gramáticas de la pregunta 2 no tienen variables inútiles?

Gracias :full_moon_with_face:

**Comment by crivero1** [comment]
No, la gramática puede traer variables inútiles. Ahora, en las clases de esta semana vimos como remover estas variables, lo cual te puede servir.  

Suerte!
